pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy

```diff
@@ -114,17 +127,37 @@ class BuildWheelsTask extends DefaultTask {
      */
     private void buildWheels(Project project, Collection<File> installables, PythonDetails pythonDetails) {
 
+        ProgressLoggerFactory progressLoggerFactory = getServices().get(ProgressLoggerFactory)
+        ProgressLogger progressLogger = progressLoggerFactory.newOperation(BuildWheelsTask)
+        progressLogger.setDescription("Building Wheels")
+        progressLogger.started()
+
         WheelExtension wheelExtension = ExtensionUtils.getPythonComponentExtension(project, WheelExtension)
         def pythonExtension = ExtensionUtils.getPythonExtension(project)
 
+        def taskTimer = new TaskTimer()
+
+        int counter = 0
+        def numberOfInstallables = installables.size()
         installables.each { File installable ->
 
             def packageInfo = PackageInfo.fromPath(installable.path)
-            def shortHand = packageInfo.version ? "${packageInfo.name}-${packageInfo.version}" : packageInfo.name
-            def messageHead = 'Preparing wheel ' + shortHand
+            def shortHand = packageInfo.version ? "${ packageInfo.name }-${ packageInfo.version }" : packageInfo.name
+
+            def clock = taskTimer.start(shortHand)
+            progressLogger.progress("Preparing wheel $shortHand (${ ++counter } of $numberOfInstallables)")
+
+            if (pythonExtension.consoleOutput == ConsoleOutput.RAW) {
+                LOGGER.lifecycle("Installing {}", shortHand)
+            }
 
             if (packageExcludeFilter.isSatisfiedBy(packageInfo)) {
-                logger.lifecycle(PythonHelpers.createPrettyLine(messageHead, "[EXCLUDED]"))
+                return
```

Comment:
Having excluded packages logged is really helpful for reasoning about conditional dependencies between Python 2 and 3 if anything goes wrong with the build. Can we get the logging back in?